spec/javascripts/support/jasmine.yml

```diff
@@ -7,7 +7,9 @@ src_files:
 css_files:
  - "support.{css,css.sass}"
 
-spec_dir: ../spec/javascripts
+spec_dir: 
+  - "../spec/javascripts"
+  - "../*/spec/javascripts"
```

Comment:
I think this new line looks confusing if someone were to see it outside the context of this PR. Is there a way to make its addition reveal your intent a little (e.g. point directly to your myengine example?)